GamesApp/GamesApp/Scenes/History/Clean/HistoryViewController.swift

```diff
@@ -23,11 +59,13 @@ class HistoryViewController: UIViewController {
     }
 }
 
-extension HistoryViewController : GameViewControllerDelegate, RPSViewControllerDelegate{
-    
-    func getBestSet(number: Int) {
+extension HistoryViewController {
+}
+
+extension HistoryViewController: HistoryDisplayLogic {
+    func displayRPSBestSet(viewModel: HistoryModels.History.ViewModel.BGSet) {
         contentView.dataStore.bestGameSetArray.remove(at: 0)
-        contentView.dataStore.bestGameSetArray.insert(.init(id: 0, bestRPSSet: number, one: 0, two: 0, three: 0, four: 0, five: 0, six: 0), at: 0)
+        contentView.dataStore.bestGameSetArray.insert(.init(id: 0, bestRPSSet: viewModel.result, one: 0, two: 0, three: 0, four: 0, five: 0, six: 0), at: 0)
         var currentSnapshot = contentView.dataSource.snapshot()
```

Comment:
ViewController owns the ContentView and must not to know about dataStore included in ContentView.
Pass the necessary data to ContentView, then into the ContentView u are to manage with passed data and configure the snapshot for dataStore.